gbalink/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
 include_directories(. ./../Runtime ${ATHENA_INCLUDE_DIR} ${JBUS_INCLUDE_DIR})
-add_executable(gbalink main.cpp)
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -ldl")
```

Comment:
What function references dl* calls? None of our user code in that target uses dynamic loader functions.